ui/de-lib/src/main/java/org/iplantc/de/client/services/DiskResourceServiceFacade.java

```diff
@@ -1,5 +1,7 @@
 package org.iplantc.de.client.services;
 
+import java.util.List;
```

Comment:
I think your IntelliJ is missing one of the code styles that makes sure `org.iplantc.*` imports come first, FYI